test/TEST-69-SHUTDOWN/TEST-69-SHUTDOWN.service

```diff
@@ -5,6 +5,8 @@ Wants=basic.target multi-user.target
 After=basic.target
 Before=getty-pre.target
 
+ConditionPathExists=/usr/bin/python3
```

Comment:
Does this mean the test won't run in the legacy setup? If so, how about running a script that simply execs true if python is not found, and the new python code if it is found?

Reply:
No in the legacy setup the test runs on the host, not in the virtual machine, so it doesn't matter if the unit is skipped. (it was already a noop before)